appcenter-auth/android/src/main/java/com/microsoft/appcenter/reactnative/auth/AppCenterReactNativeAuthModule.java

```diff
@@ -27,4 +39,54 @@ public AppCenterReactNativeAuthModule(Application application) {
     public String getName() {
```

Comment:
Add a `@Nonnull` annotation to `getName()` to avoid the Android Studio warning.

Reply:
Done